src/noit_metric_tag_search.c

```diff
@@ -214,6 +262,7 @@ void var_graphite_free(void *vi) {
   struct graphite_impl *g = vi;
   if(g == NULL) return;
   for(int i=0; i<g->count; i++) {
+    mtev_lfu_release_f(re_matcher_free, g->results[i].tok);
```

Comment:
not needed, ``re_matcher_free`` was already set as the "free" function when the cache was created

```suggestion
    mtev_lfu_release((g->results[i].tok);
```

Reply:
lfu_release take the LFU itself as the first argument... which could have been destroyed by the time the thread using this AST ultimaltely releases it.